### PR TITLE
Fix osm importer projection

### PR DESCRIPTION
**Pull request by DavidMansolino**
Document new argument
Linked with omichel/webots#4108

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: automobile/scenario-creation-tutorial.md [hunk 1]
```diff
@@ -20,14 +20,16 @@ This importer uses splines to improve and smooth the path of the roads, unfortun
...
 
 ```
  * map centered with this offset: x_offset_value,y_offset_value.
  * reference coordinates: latitude_value,longitude_value.
+ * projection used: 'projection parameters'.
 ```
 
-It is important that you keep these offset and reference coordinates since you will need to reuse them to generate traffic with SUMO.
+It is important that you keep these offset and projection since you will need to reuse them to generate traffic with SUMO.
+The projection parameters should looks like: `'+proj=utm +north +zone=32 +lon_0=6.505000 +lat_0=46.511000 +x_0=0 +y_0=0 +ellps=WGS84 +units=m +no_defs'`
```
Comment (omichel) [inline automobile/scenario-creation-tutorial.md]: looks &rarr; look

**Comment by DavidMansolino** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**